rules-tests/DowngradePhp70/Rector/Expression/DowngradeDefineArrayConstantRector/Fixture/skip_first_class_callable.php.inc

```diff
@@ -0,0 +1,8 @@
+<?php
+
+namespace Rector\Tests\DowngradePhp70\Rector\Expression\DowngradeDefineArrayConstantRector\Fixture;
+
+function skipFirstClassCallable()
+{
+    $firstClassCallable = define(...);
```

Comment:
this will show error when enable zend.assertion in CI https://www.php.net/manual/en/ini.core.php#ini.zend.assertions

Reply:
Here the example setting assertion setting in console:

```
# disable
php > ini_set('zend.assertions', 0);
php > assert(true === false);

# enable
php > ini_set('zend.assertions', 1);
php > assert(true === false);
PHP Warning:  Uncaught AssertionError: assert(true === false) in php shell code:1
Stack trace:
#0 php shell code(1): assert(false, 'assert(true ===...')
#1 {main}
  thrown in php shell code on line 1

Warning: Uncaught AssertionError: assert(true === false) in php shell code:1
Stack trace:
#0 php shell code(1): assert(false, 'assert(true ===...')
#1 {main}
  thrown in php shell code on line 1
``

Reply:
Ok, `isFirstCallable()` already called on `shouldSkip()` method, so it works

https://github.com/rectorphp/rector-downgrade-php/blob/86a937c9c1b31ed702070cb39f2ee2d4748f545b/rules/DowngradePhp70/Rector/Expression/DowngradeDefineArrayConstantRector.php#L86-L88